lib/audit/checksum.rb

```diff
@@ -51,7 +51,7 @@ def self.validate_druid(druid)
     start_msg = "#{Time.now.utc.iso8601} CV validate_druid starting for #{druid}"
     puts start_msg
     Rails.logger.info start_msg
-    pres_copies = PreservedCopy.joins(:preserved_object).where(preserved_objects: { druid: druid })
+    pres_copies = PreservedCopy.by_druid(druid)
```

Comment:
OMG this is awesome.  Would you please make a ticket to look for other places in our code and tests where we could use this and simplify the code?

Reply:
#644